popup.js

```diff
@@ -1,38 +1,46 @@
 let active = document.getElementById('active');
+const reloadInput = document.getElementById('reload-int-input');
```

Comment:
Can you also please make "active" a constant too?

Reply:
Done